.github/ISSUE_TEMPLATE.md

```diff
@@ -3,7 +3,24 @@
 #### Description of the problem
 
 <!-- If you can, it's helpful to provide self-contained example of some code
-that reproduces the bug. If not, a link to a larger example is also helpful. -->
+that reproduces the bug (you can uncomment the @coqbot: minimize block below).
+If not, a link to a larger example is also helpful. -->
+
+<!-- @coqbot: minimize
+```
```

Comment:
Does 
```suggestion
```bash
```
work too?

Reply:
The regex would need to be altered to ignore any non-space characters immediately following the triple backticks.  I think it would be a good idea to make this change.